src/Atoms/Link/Link.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Box, Link as BaseLink } from '@material-ui/core'
+
+const Link = ({ children, ...props }) => (
+  <Box fontFamily="fontFamily">
+    <BaseLink style={{ color: '#337AB7' }} {...props}>
```

Comment:
No theming?

Reply:
We wanted to use a new Mui theme palette (`link`) but sadly `useTheme` hook returns the OG Mui theme, not ours, so this is just a quickfix 🤷‍♂️

Reply:
Sadly, using :

```diff
diff --git a/.storybook/themes/mui-rg6-theme.js b/.storybook/themes/mui-rg6-theme.js
index 0725ab1..d0930e9 100644
--- a/.storybook/themes/mui-rg6-theme.js
+++ b/.storybook/themes/mui-rg6-theme.js
@@ -10,6 +10,7 @@ export default createMuiTheme({
     info:      { main: '#337AB7' },
     warning:   { main: '#F0AD4E' },
     error:     { main: '#D9534F' },
+    link:      { main: '#5CB85C', contrastText: 'white' },
   },
   typography: {
     fontFamily: '"Helvetica Neue", Helvetica, Arial, sans-serif',
diff --git a/src/Atoms/Link/Link.jsx b/src/Atoms/Link/Link.jsx
index 1c58652..3c927e9 100644
--- a/src/Atoms/Link/Link.jsx
+++ b/src/Atoms/Link/Link.jsx
@@ -3,7 +3,7 @@ import { Box, Link as BaseLink } from '@material-ui/core'

 const Link = ({ children, ...props }) => (
   <Box fontFamily="fontFamily">
-    <BaseLink style={{ color: '#337AB7' }} {...props}>
+    <BaseLink color="link" {...props}>
       { children }
     </BaseLink>
   </Box>
```

generates this error :

```
checkPropTypes.js:20 Warning: Failed prop type: Invalid prop `color` of value `link` supplied to `ForwardRef(Link)`, expected one of ["initial","inherit","primary","secondary","textPrimary","textSecondary","error"].
```

Reply:
Without `fontFamily` :

![image](https://user-images.githubusercontent.com/717869/80106051-a1c80a00-8579-11ea-928e-43fe62b2f566.png)

With `fontFamily` :

![image](https://user-images.githubusercontent.com/717869/80106114-b73d3400-8579-11ea-841c-8fec1431dee4.png)